QuickReference.ipynb

```diff
@@ -0,0 +1,402 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 0. Setup\n",
+    "Please make sure your environment is set up according to the instructions here: https://github.com/NASA-NAVO/aas_workshop_2020_winter/blob/master/00_SETUP.md\n",
+    "\n",
+    "Ensure you have the latest version of the workshop material by updating your environment:\n",
+    "TBD"
```

Comment:
Did you intent to leave this TBD? If so, when will we fill it in?

Reply:
Yes, I'd like to think about it a little more, and patch it on Saturday.